clang/lib/CIR/CodeGen/CIRGenItaniumCXXABI.cpp

```diff
@@ -360,6 +360,29 @@ class CIRGenItaniumCXXABI : public CIRGenCXXABI {
   classifyRTTIUniqueness(QualType CanTy, cir::GlobalLinkageKind Linkage) const;
   friend class CIRGenItaniumRTTIBuilder;
 };
+
+class CIRGenARMCXXABI : public CIRGenItaniumCXXABI {
+public:
+  CIRGenARMCXXABI(CIRGenModule &CGM) : CIRGenItaniumCXXABI(CGM) {}
+  // TODO: When implemented, /*UseARMMethodPtrABI=*/true,
```

Comment:
We usually go for `TODO(cir)` to differentiate from `TODO`s we copy-n-paste from OG codegen